css/style.css

```diff
@@ -566,6 +583,69 @@ body {
 .title-navbar:hover {
     color: var(--hover-color);
 }
+.drawer{
+    margin:auto;
+}
+
+.drawer-container {
+    padding: 1rem;
+    background-color: var(--background-color);
+    width: 250px;
```

Comment:
Why px here? you can convert it to rem right?

Reply:
I will change it